src/client.rs

```diff
@@ -390,6 +409,49 @@ impl HttpClientBuilder {
         self
     }
 
+    /// Set a custom SSL/TLS CA certificate bundle to use for all client connections.
+    ///
+    /// The default value is none.
+    ///
+    /// Note: for Windows, setting `no_revoke(true)` might also be nessery.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use isahc::config::*;
+    /// # use isahc::prelude::*;
+    /// #
+    /// let client = HttpClient::builder()
+    ///     .ca_certificate("ca.pem".into())
+    ///     .build()?;
+    /// # Ok::<(), isahc::Error>(())
+    /// ```
+    pub fn ca_certificate(mut self, ca_path: PathBuf) -> Self {
+        self.defaults.insert(CACertificatePath { path: ca_path });
+        self
+    }
+
+    /// Disable certificate revocation checks for those SSL backends where such behavior is present.
+    /// This option is only supported for Schannel (the native Windows SSL library),
+    ///
+    /// The default value is false.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use isahc::config::*;
+    /// # use isahc::prelude::*;
+    /// #
+    /// let client = HttpClient::builder()
+    ///     .no_revoke(true)
+    ///     .build()?;
+    /// # Ok::<(), isahc::Error>(())
+    /// ```
+    pub fn no_revoke(mut self, on: bool) -> Self {
```

Comment:
I'd give this a more descriptive name that makes it clear that this is related to SSL. Most of the SSL-related methods begin with `ssl_`, so how about `ssl_no_revoke`?